desktop/src/main/kotlin/io/spine/examples/pingh/desktop/AppState.kt

```diff
@@ -62,7 +63,29 @@ internal class AppState(settings: SystemSettings) {
 
     init {
         val notificationSender = TrayNotificationSender(composeTray) { !window.isShown }
-        app = PinghApplication(notificationSender)
+        val properties = loadProperties()
+        app = PinghApplication.builder()
+            .withAddress(properties.getProperty("server.address"))
+            .withPort(properties.getProperty("server.port").toInt())
+            .with(notificationSender)
+            .build()
+    }
+
+    /**
+     * Loads server properties from resource folder.
+     */
+    private fun loadProperties(): Properties {
+        val properties = Properties()
+        javaClass.getResourceAsStream("/config/server.properties").use {
+            properties.load(it)
+        }
+        check(properties.containsKey("server.address")) {
+            "The Pingh server address must be provided in the configuration file."
```

Comment:
How the readers of this message are going to understand where the property file is expected?